config/locales/time/fr.yml

```diff
@@ -1,5 +1,22 @@
 ---
 fr:
+  date:
+    month_names:
+    - 
+    - janvier
+    - février
+    - mars
+    - avril
+    - mai
+    - juin
+    - juillet
+    - août
+    - septembre
+    - octobre
+    - novembre
+    - décembre
   time:
+    am: NOT TRANSLATED YET
     formats:
       event_timestamp: "%B %e, %Y à %-l:%M %p"
```

Comment:
In French the day comes before the month, as in "27 Décembre, 2017". Can we change the event_timestamp to `%e %B, %Y à %-l:%M %p`?

Reply:
yes.  I can put the format in the locale file

Reply:
Ah I see it already is in the locale file....I'll apply the change you mentioned